src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -665,6 +660,8 @@ export class ServerView extends DirMixin(PolymerElement) {
        */
       this.myConnection = null;
       this.totalInboundBytes = 0;
+      this.inboundByesUnit = 'B';
```

Comment:
We should either leave this empty, or set it according to the language.
Maybe you set those fields on the connected callback, or when the language is set.